components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/model/FilterData.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update to new license header. year and LLC change
Also update the LLC change in other files that you have modified